controller/cmd/destination/main.go

```diff
@@ -56,7 +58,11 @@ func Main(args []string) {
 	go func() {
 		log.Infof("starting admin server on %s", *metricsAddr)
 		if err := adminServer.ListenAndServe(); err != nil {
-			log.Errorf("failed to start destination admin server: %s", err)
+			if errors.Is(err, http.ErrServerClosed) {
+				log.Infof("destination admin server status is shutdown or close: %s", err)
```

Comment:
Looking at the docs, the error printed for graceful shutdown (i.e. `http.ErrServerClosed`) is something like:

```
"http: Server closed"
```

[ref](https://pkg.go.dev/net/http#ErrServerClosed)

Can we maybe change this to:

```
log.Infof("shutting down admin server on %s", *metricsAddr)
```

Otherwise our log message will look like this:

```
destination admin server status is shutdown or close: http: Server closed
```